src/contracts/identity.test.ts

```diff
@@ -125,4 +128,36 @@ describe("identity", () => {
       expect(authorized).toBe(true);
     });
   });
+
+  describe("#createAndRegisterBeaconProxy", () => {
+    const handle = "flarp";
+    const fakeAddress = "0x1Ea32de10D5a18e55DEBAf379B26Cc0c6952B168";
+
+    it("returns a Contract Transaction that can be resolved into a DSNP Id", async () => {
+      const transaction = await createAndRegisterBeaconProxy(fakeAddress, handle);
+
+      const receipt = await transaction.wait(1);
+
+      expect(getIdFromReceipt(receipt).toNumber()).toBeGreaterThan(999);
```

Comment:
Just curious - why is this expectation greater than 999?

Reply:
The registry contract starts ids at 1000. So we know it shouldn't be less, but we don't know much more than that. I added a comment to this effect.